plugins/modules/telegram.py

```diff
@@ -16,7 +16,7 @@
  - "Artem Feofanov (@tyouxa)"
  - "Nikolai Lomov (@lomserman)"
 
-short_description: module for sending notifications via telegram
+short_description: Module for sending notifications via telegram
```

Comment:
How about:
```suggestion
short_description: Send notifications via telegram
```
After all it should be clear from context that this is a module.